README.md

```diff
@@ -8,7 +8,7 @@ Uploads continue in the background, even after a page transition. In other words
 
 ## Compatibility
 
-* Ember.js v2.18 or above
+* Ember.js v3.16 or above
 * Ember CLI v2.13 or above
```

Comment:
Should we update the supported Ember CLI version also?

Reply:
That's actually straight out of the [latest blueprint](https://github.com/ember-cli/ember-addon-output/blob/v3.26.0/README.md). I guess very modern addons can still run on a super-old `ember-cli` 🤷

Reply:
Ok sure thing 🤷